src/ethereum_spec_tools/evm_tools/t8n/env.py

```diff
@@ -242,7 +242,6 @@ def read_block_hashes(self, data: Any, t8n: "T8N") -> None:
         self.parent_hash = None
         if t8n.fork.is_after_fork("ethereum.prague"):
             self.parent_hash = Hash32(hex_to_bytes(data["parentHash"]))
-            return
```

Comment:
Confirming this is related to EIP-7251

Reply:
This is basically a port of the changes to EIP-2935. Although not directly related to EIP-7251, this is essential to make the tests work with the latest test fixtures.